banners/desktop/banner_ctrl.ts

```diff
@@ -59,7 +59,8 @@ app.use( DynamicTextPlugin, {
 	date: new Date(),
 	formatters: localeFactory.getFormatters(),
 	impressionCount,
-	translator
+	translator,
+	urgencyMessageDaysLeft: 45
```

Comment:
Is this a leftover from some experiment?

Reply:
No, the `campaignDaySentence` shows a more urgent message if we're within `N` days of the end date. It defaults to 10 but they wanted to activate it early and 45 was the random number I picked that was greater than the current days left when it was activated.